### ISSUE TITLE
Port Workers tab does not implement drag selection or ctrl-A, etc.

### ISSUE THREAD

**Issue by RHolton** [widelands/widelands#5577]
When I go to a port, I can select multiple wares using a left-mouse-button drag. I thought things like Ctrl-A would also work, but maybe I'm wrong.

However, when I go to the Workers tab, I cannot use the left-mouse-button drag. 

Running Widelands-daily-msvc-Release-x64 ec44609@master, on  Windows 11.

**Comment by Noordfrees** [comment]
I cannot reproduce. Are you sure you were using the second tab (workers in warehouse) and not the last tab (workers waiting for a ship)?

![grafik](https://user-images.githubusercontent.com/48293446/195286004-efe3e113-4776-4ec5-ac26-3f1c9104405e.png)

Supporting Ctrl+A in ware lists would be a new feature.
